ecommerce-product-page-main/styled.css

```diff
@@ -200,98 +197,65 @@ body{
     background-color: hsl(26,100%,55%);
 }
 
-
-
-img.space{
-    margin-top: 12px;
-    width: 130px;
-}
-
-ul.list{
-
+.list {
     display: flex;
-    list-style: none;
+    text-decoration: none;
+    gap: 10px;
+    align-items: center;

```

Comment:
In your line 138 you used the "justify-content: space-around" while here you're using "align-items". Seeing how the list class is a content of the container class wont this new css command over ride the one in line 138?

Reply:
justify content and align items are two different css properties. They wont override eachother but instead work together